src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -323,6 +324,26 @@ public function xdebugAction(): Response
         return new Response($xdebugInfo, 200, ['Content-Type' => 'text/html']);
     }
 
+    /**
+     * Return the JetBrainMono font.
+     *
+     * @throws NotFoundHttpException
+     */
+    public function downloadFontAction(string $font): Response
```

Comment:
I'd rename the argument as `$fontFamilyName` to be more precise.

Reply:
Not sure if this is open to discussion.. but i find this block more readable with "`font`" (ie: a _file_) than "`fontFamilyName`" (`font_family_name` ?)

```twig
{# simplified #} 
@font-face {
    font-family: 'JetBrains Mono';
    font-style: normal;
    font-weight: 100 900;
    src:
        local('JetBrains Mono'),
        local('JetBrainsMono'),
        url('{{ url('_profiler_download_font', {font: 'JetBrainsMono-Variable'}) }}') format('woff2');
}
```

I'd keep the variable name "$font", but i'd rename '`JetBrainsMono`' into '`JetBrainsMono-Variable`' 

What do you think ?

Reply:
Sorry for the nitpick, but I think the variable name should be precise about its contents. To me:

* `$font`, the binary data of a single font file (e.g. `0001010100...`)
* `$fontFile` or `$fontFilePath`, the path of some binary font file (e.g. `./fonts/JetBrainsMono-ExtraLightItalic.woff2`
* `$fontName`, the name of a font file (e.g. `JetBrainsMono-MediumItalic.otf`)
* `$fontFamily`, either a ZIP file with all the fonts that belong to the family or the path to that file (e.g. `/uploads/fonts/JetBrainsMono-2.304.zip`)
* `$fontFamilyName`, only the name of the font family (as used e.g. by CSS) (e.g. `JetBrainsMono`)

Reply:
but was is passed here is the basename of the font file. It is not the font family name.

Reply:
Correct! Sorry. Then, the variable should be `$fontFileName`, right?

We use it here, so it's not a path, URL, or binary file. It's only the `fileName`:

```php
$file = \dirname(__DIR__).'/Resources/fonts/'.$font.'.woff2';
```

Reply:
Yes! Thank you ... and sorry for insisting so much on this 🙏

Reply:
Don't be! And thank _you_ for the feedbacks :)